tailwind.config.js

```diff
@@ -6,7 +6,10 @@ module.exports = {
     extend: {
       gridTemplateColumns: {
         'chips-sheet': 'repeat(auto-fit, minmax(90px, min-content))',
-      }
+      },
+      width: {
+        '10p': '10%',
```

Comment:
according to existing tailwind classed in their documentation we have to name value like this - 1/10